src/main/java/com/puppycrawl/tools/checkstyle/utils/CommonUtils.java

```diff
@@ -467,4 +467,31 @@ public static String getFileExtension(String fileNameWithExtension) {
         }
         return extension;
     }
+
+    /**
+     * Checks whether the given string is a valid identifier.
+     * @param str A string to check.
+     * @return true when the given string contains valid identifier.
+     */
+    public static boolean isIdentifier(String str) {
+        boolean isIdentifier = !str.isEmpty() && Character.isJavaIdentifierStart(str.charAt(0));
```

Comment:
@rnveach, Is the following code appropriate? I wrapped the loop with the `if` statement and removed `isIdentifier` from the loop condition. As for me, it is less clear, but If you're fine with the changes, I'll move them into PR.

```
public static boolean isIdentifier(String str) {
  boolean isIdentifier = false;
  if (!str.isEmpty() && Character.isJavaIdentifierStart(str.charAt(0))) {
    isIdentifier = true;
    for (int i = 1; i < str.length(); i++) {
      if (!Character.isJavaIdentifierPart(str.charAt(i))) {
        isIdentifier = false;
        break;
      }
    }
  }
  return isIdentifier;
}
```